simplesamlphp/simplesamlphp/201709-01.yaml

```diff
@@ -0,0 +1,7 @@
+title:     Cross Site Scripting (XSS) in the consentAdmin module
+link:      https://simplesamlphp.org/security/201709-01
+branches:
+    1.14.x:
+        time:     2017-08-25 11:35:00
+        versions: ['>=1.14.0', '<1.14.16']
```

Comment:
The blog post says that *all* versions until 1.14.15 (included) are affected. Your advisory here says that 1.13 releases are not affected.

Reply:
You are right. I'll just send another pr. :+1: